NeosModLoader/AssemblyLoader.cs

```diff
@@ -38,6 +37,7 @@ internal static class AssemblyLoader
                     Logger.ErrorInternal($"Error enumerating ${dirName} directory:\n{e}");
                 }
             }
+            Array.Sort(assembliesToLoad, (a, b) => string.CompareOrdinal(a, b));
```

Comment:
...Though wouldn't this throw if assembliesToLoad is never initialized...?

Reply:
It would. Somehow that got moved down out of the try...catch. I moved it back in now.